ods_ci/tests/Resources/Files/triton/kserve-triton-tensorflow-output.json

```diff
@@ -0,0 +1 @@
95219850540161,0.05419670417904854,1.075439214706421,0.5701701045036316,1.9159470796585083,1.4319592714309692,0.3080004155635834,-0.20652148127555847,2.0885848999023438,-2.9661037921905518,0.40664154291152954,1.5939422845840454,1.3975993394851685,-1.5359636545181274,1.4335156679153442,-0.6088399887084961,-0.6214863657951355,-2.9783685207366943,-2.9415695667266846,-2.0613062381744385,-1.9683763980865479,1.9645687341690063,2.1087796688079834,-0.6631877422332764,-1.5008337497711182,2.1004319190979004,-0.29084524512290955,-3.970994234085083,0.774614691734314,-0.10501639544963837,-0.7073017358779907,0.6008784770965576,1.6123569011688232,-0.13200010359287262,0.7118274569511414,1.2247412204742432,0.8188934326171875,-0.0008978259284049273,1.7649238109588623,-0.07088223844766617,-1.4360523223876953,-1.9582669734954834,0.1315285712480545,-3.0323493480682373,0.4263181984424591,4.756340503692627,4.023977279663086,1.9825353622436523,0.09579263627529144,1.3818793296813965,0.9103924632072449,1.118933916091919,0.4180070459842682,-0.5536695718765259,0.5137543082237244,2.8405511379241943,1.625567078590393,3.4743645191192627,1.913105845451355,1.176734447479248,0.5754279494285583,0.18591882288455963,-0.3994935154914856,1.9705591201782227,2.6490659713745117,-0.759437084197998,-0.23889786005020142,-0.900497317314148,-1.4153908491134644,0.541402280330658,-0.2134995460510254,0.24751678109169006,0.6561787724494934,0.35915306210517883,3.5687508583068848,1.4349572658538818,-1.9755446910858154,-2.1722967624664307,-1.7886747121810913,-3.6240921020507812,-1.7883633375167847,-2.1531994342803955,-0.05177495628595352,-1.1518805027008057,-2.2381911277770996,-0.2777620553970337,-2.7935473918914795,-0.985107421875,-1.5737550258636475,-2.8648784160614014,-0.7926676273345947,-1.9847651720046997,-1.9565452337265015,-2.3109192848205566,-2.170433759689331,-2.046128273010254,-1.7884387969970703,-3.1766295433044434,-0.7652154564857483,-2.8208019733428955,-1.8925821781158447,-0.3248683512210846,-0.6802529096603394,-0.030026616528630257,0.22050660848617554,-1.0655189752578735,-1.0855458974838257,0.9682175517082214,-2.5945935249328613,-1.4845701456069946,-0.9860842227935791,-0.2501584589481354,-2.21254825592041,-0.9282965660095215,-0.5894380211830139,-2.2491068840026855,-1.3508793115615845,-2.069303274154663,-1.6948262453079224,0.9848254323005676,1.2485548257827759,0.9365651607513428,-1.2274106740951538,-1.7764509916305542,0.5567451119422913,-1.3579906225204468,-2.8065943717956543,-0.5851900577545166,-0.13392241299152374,-0.6354706883430481,1.0494153499603271,-0.35455286502838135,-2.4247593879699707,0.09886763244867325,0.014342203736305237,-1.8393536806106567,-1.2256815433502197,-0.19250613451004028,-1.3938517570495605,-2.2999706268310547,-0.45072028040885925,-1.0265564918518066,-0.11560125648975372,-0.9679169058799744,-1.5078604221343994,-3.00948166847229,-2.543466091156006,-3.6631100177764893,-2.847332715988159,-2.45154070854187,-1.5960808992385864,-1.6782920360565186,2.192574977874756]}]}
```

Comment:
maybe better to add a reference to the model name in the input and output filenames